### PR TITLE
Update support for several browsers and HTML viewer

### PR DESCRIPTION
**Pull request by stevenlele**
This patch adds the Chinese version of Via browser ([`mark.via`](https://www.coolapk.com/apk/mark.via)) and updates the uri view id for both Chinese and Google Play version.

I also added HTML viewer (`com.android.htmlviewer`) since my system used it to display the WiFi log-in page. **It does NOT have a url bar so I did NOT add it to `AccessibilityHelpers.cs`.**

### CODE REVIEW

**Comment by stevenlele** [comment]
Added the following Chinese browsers:
- Browser [`com.mmbox.browser`](https://www.coolapk.com/apk/com.mmbox.browser)
- X Browser [`com.mmbox.xbrowser`](https://www.coolapk.com/apk/com.mmbox.xbrowser)
- Magic Browser [`com.z28j.feel`](https://www.coolapk.com/apk/com.z28j.feel)

**Comment by stevenlele** [comment]
@mportune-bw Could you please have a look at this PR? Thanks a lot!

File: src/Android/Autofill/AutofillHelpers.cs [hunk 1]
```diff
@@ -41,6 +41,7 @@ public static class AutofillHelpers
             "com.amazon.cloud9",
             "com.android.browser",
             "com.android.chrome",
+            "com.android.htmlviewer",
```
Comment (mportune-bw) [inline src/Android/Autofill/AutofillHelpers.cs:44]: Does this need to be added to `SupportedBrowsers` above as well?

Comment (stevenlele) [reply src/Android/Autofill/AutofillHelpers.cs:44]: idk, as I've said, it does not have a url bar, so I doubt if the accessibility service will work on this.

**Review by mportune-bw — APPROVED** [review]

File: src/Android/Autofill/AutofillHelpers.cs (hunk 1 above)
Comment (contribucious) [reply src/Android/Autofill/AutofillHelpers.cs:44]: **[2020/12]** Precision now added in `SupportedBrowsers` about the non-presence of this entry. See PR https://github.com/bitwarden/mobile/pull/1187.

**Pull request merged**